src/main/java/edu/kis/powp/jobs2d/command/transformations/TransformationManager.java

```diff
@@ -0,0 +1,211 @@
+package edu.kis.powp.jobs2d.command.transformations;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import edu.kis.powp.jobs2d.command.CommandCoordinatesVisitor;
+import edu.kis.powp.jobs2d.command.DriverCommand;
+import edu.kis.powp.jobs2d.command.ICompoundCommand;
+import edu.kis.powp.jobs2d.command.ImmutableComplexCommand;
+import edu.kis.powp.jobs2d.command.OperateToCommand;
+import edu.kis.powp.jobs2d.command.SetPositionCommand;
+import edu.kis.powp.jobs2d.command.line.Line2d;
+
+public class TransformationManager {
+	
+	public ICompoundCommand moveCommand(CommandCoordinatesVisitor visitor, int x, int y) {
+
+		List<Line2d> lines = visitor.getAllCommandsCoordinates();
+
+		moveToPosition(lines, x, y);
+
+		List<DriverCommand> commandList = buildCommandList(lines);
+
+		return new ImmutableComplexCommand(commandList);
+	}
+
+	public ICompoundCommand flipHorizontalCommand(CommandCoordinatesVisitor visitor) {
+
+		List<Line2d> lines = visitor.getAllCommandsCoordinates();
+
+		for(int i = 0; i < lines.size(); i++) {
+			lines.get(i).setStartPosX(-lines.get(i).getStartPosX());
+			lines.get(i).setEndPosX(-lines.get(i).getEndPosX());
+		}
+
+		List<DriverCommand> commandList = buildCommandList(lines);
+
+		return new ImmutableComplexCommand(commandList);
+	}
```

Comment:
All methods share some code (duplication). The code is not OCP. Adding new transformation requires modification in this class (containing other transformations).